pkg/solana/client.go

```diff
@@ -52,8 +53,16 @@ func NewClient(spec OCR2Spec, logger Logger) *Client {
 		ctxInterval = 2 * pollInterval
 	}
 
+	// parse tx context, if errors use defaultStaleTimeout
+	txTimeout, err := time.ParseDuration(spec.TxTimeout)
+	if err != nil {
+		logger.Warnf("could not parse tx context duration using default 1m")
```

Comment:
Maybe include the err or string that failed to parse?

Reply:
good idea, i also implemented it for parsing the other 2 timeouts as well